projects/step-core/src/lib/modules/tree/components/tree/tree.component.html

```diff
@@ -18,7 +18,7 @@
   >
     <button mat-button mat-icon-button disabled></button>
     <div class="node-content" (contextmenu)="openContextMenu($event, node.id)" cdkDragHandle>
-      <step-icon [name]="node._class | artefactIcon"></step-icon>
+      <step-icon [name]="node.icon" [class]="node.iconClassName"></step-icon>
```

Comment:
What is the class doing with the step-icon?

Reply:
For execution tree it is required to highlight the the icon with status color. To do it, I've added an ability to tree component that allows to customize icon's styles with additional css class